packages/near-fast-auth-signer/src/components/SignMultichain/types.ts

```diff
@@ -0,0 +1,57 @@
+import { KeyPair } from 'near-api-js';
+
+export interface DerivationPathDeserialized {
+  asset: 'ETH' | 'BNB' | 'BTC';
+  domain?: string;
+}
+
+export interface MultichainExecutionResponse {
```

Comment:
I think some of those types you can clean up. Like this is only used here. Please also check the other types